test/e2e/cypress/integration/recents-panel.spec.ts

```diff
@@ -0,0 +1,19 @@
+describe('Recents panel', () => {
+    beforeEach(() => {
+      cy.clearLocalStorage();
+      cy.login();
+      cy.visit('/app/recents');
+    });
+
+    it('Should list recents', () => {
+      cy.contains('example.txt');
+    });
+
+    it('Should get link', () => {
+      cy.get('button.file-list-item-actions-button').click();
+      cy.get('a').contains('Get link').click();
+      cy.get('button.file-list-item-actions-button').click();
+      cy.contains('Delete link');
```

Comment:
Here we also want to make sure the link is deleted, perhaps checking for "link deleted" and the link logo no longer there could be a good idea

Reply:
There is no task to remove the link, it is the way to check that the link has been generated but I will find another approach to do it.

Reply:
comment added